src/lsp/cobol_lsp/lsp_completion.ml

```diff
@@ -162,16 +162,24 @@ let procedures_proposal ~filename pos group =
   match Lsp_lookup.last_cobol_unit_before_pos ~filename pos group with
   | None -> []
   | Some cu ->
-    let paragraph_name (paragraph:Cobol_unit.Types.procedure_paragraph with_loc) =
-      Option.map Cobol_common.Srcloc.payload ~&paragraph.paragraph_name
+    let to_string_with_type
+    typ (paragraph:Cobol_unit.Types.procedure_paragraph with_loc) =
```

Comment:
```suggestion
    let to_string_with_type typ paragraph =
```
(no need for the annotation if that wrecks formatting. `paragraph` is explicit enough in this context. (Note you can `open Cobol_unit.Types` if needed.)